plottable.css

```diff
@@ -158,3 +158,8 @@ svg.plottable {
   stroke: #CCC;
   stroke-width: 1px;
 }
+
+.plottable .segment-plot line {
+  stroke: #555;
+  stroke-width: 1px;
```

Comment:
Do we have design sign-off on this? I would instead default to the same color other `Plot`s do, using an `attr()` call in the `constructor`.